CMakeLists.txt

```diff
@@ -28,8 +28,8 @@ option (PIO_USE_MALLOC       "Use native malloc (instead of bget package)"  OFF)
 option (PIO_MICRO_TIMING     "Enable internal micro timers"                 OFF)
 option (PIO_SAVE_DECOMPS     "Dump the decomposition information"           OFF)
 option (WITH_PNETCDF         "Require the use of PnetCDF"                   ON)
-option (WITH_NETCDF          "Require the use of NetCDF"                    ON)
-option (WITH_ADIOS           "Require the use of ADIOS"                     OFF)
+option (WITH_NETCDF          "Require the use of NetCDF"                    OFF)
+option (WITH_ADIOS           "Require the use of ADIOS"                     ON)
```

Comment:
These defaults shouldn't be changed. You can change it to test ADIOS by setting it at configure time.

Reply:
Done. Will create a new pull request.